application/views/admin/expressions/test/survey_logic_file.php

```diff
@@ -32,4 +32,27 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+<style>
```

Comment:
Can not be included in style file ?

Reply:
As this is kind of a workaround, we thought this could be a good place.

Reply:
workaround for bad css ;)